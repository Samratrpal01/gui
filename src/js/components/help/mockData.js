export const helpProps = {
  docsVersion: undefined,
  isHosted: false,
  menderVersion: 'master',
  isEnterprise: true,
  tenantCapabilities: {},
  tokens: [],
  versions: {
    repos: {
      mender: '3.3.0',
      'mender-artifact': '3.8.0',
      'mender-binary-delta': '1.3.1',
      'mender-cli': '1.8.0',
      'mender-configure-module': '1.0.4',
      'mender-connect': '2.0.2',
      'mender-convert': '3.0.0',
      'mender-gateway': '1.0.0',
      'monitor-client': '1.2.0'
    },
    releaseDate: '01-01-2003'
  }
};
