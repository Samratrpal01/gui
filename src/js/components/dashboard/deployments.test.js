import React from 'react';
import { Provider } from 'react-redux';
import thunk from 'redux-thunk';
import configureStore from 'redux-mock-store';

import { defaultState, undefineds } from '../../../../tests/mockData';
import { render } from '../../../../tests/setupTests';
import Deployments from './deployments';

const mockStore = configureStore([thunk]);

describe('Deployments Component', () => {
  let store;
  beforeEach(() => {
    store = mockStore({ ...defaultState });
  });

  it('renders correctly', async () => {
    const { baseElement } = render(
      <Provider store={store}>
        <Deployments clickHandle={jest.fn} />
      </Provider>
    );
    const view = baseElement.firstChild.firstChild;
    expect(view).toMatchSnapshot();
    expect(view).toEqual(expect.not.stringMatching(undefineds));
  });
});
