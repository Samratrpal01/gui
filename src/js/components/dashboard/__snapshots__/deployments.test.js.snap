// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Deployments Component renders correctly 1`] = `
<div
  class="undefined deployments margin-bottom-large"
>
  <div
    class="loaderContainer"
  >
    <div
      class="  loader"
    >
      <span
        class="dot dot_1"
      />
      <span
        class="dot dot_2"
      />
      <span
        class="dot dot_3"
      />
      <span
        class="dot dot_4"
      />
    </div>
  </div>
</div>
`;
